### ISSUE TITLE
Script deletes all files in output directory 

### ISSUE THREAD

**Issue by CamelCaseCam** [vfdev-5/Video2Frames#2]
If you have any files in the output directory, it deletes them.

**Comment by vfdev-5** [comment]
Yes, this is a bit dangerious behaviour but it is currently like that : https://github.com/vfdev-5/Video2Frames/blob/bed70448926b855d837eee806cc9cc197bb32542/video2frames.py#L24-L27

What would you like to suggest ?

**Comment by CamelCaseCam** [comment]
1) Rewrite it so it doesn't delete the folder 
Or 
2) Put a warning in the Readme so people don't lose files

**Comment by vfdev-5** [comment]
I see, if you would like to send a PR with a fix, 1 or 2, it can be helpful
